themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/theme.scss

```diff
@@ -295,6 +295,12 @@ h6 {
       line-height: .7em;
       margin-left: -50px;
     }
+    &.blockquote-no-pseudo-content {
+      padding-left: 30px;
+      &:before {
+        content: none;
+      }
+    }
```

Comment:
I'm not familiar with scss, I'm guessing the `&.blockquote-no-pseudo-content` means you're creating a new class. Then the `&:before` is inheriting stuff from the `&:before` above? I'm guessing this class isn't used anywhere yet and will get added later?

I previewed the site on NationBuilder using the `Ranked Choice Voting for Texas` theme and it seemed to run so will approve it.

Reply:
Yes, it's a new class and selecting for it here in combination with the enclosing selector. The &:before is for overriding the &:before specified in the enclosing selector. I'm using this new class on Donate test pages.